channels/events/events.go

```diff
@@ -49,6 +51,10 @@ func (cce createChannelEvent) Encode() (map[string]interface{}, error) {
 		"roles_provisioned": cce.rolesProvisioned,
 		"status":            cce.Status.String(),
 		"created_at":        cce.CreatedAt,
+		"domain":            cce.DomainID,
+		"user_id":           cce.UserID,
+		"token_type":        cce.Type.String(),
+		"super_admin":       cce.SuperAdmin,
```

Comment:
Do we need super admin info?

Reply:
Arvindh had mentioned I add it. I am not yet sure where we it would be used